core/src/main/java/com/linecorp/armeria/server/http/auth/OAuth1AToken.java

```diff
@@ -0,0 +1,63 @@
+package com.linecorp.armeria.server.http.auth;
+
+import java.util.Map;
+
+/**
+ * The bearer token of <a href="https://oauth.net/core/1.0a/#rfc.section.6.3.1">OAuth 1.0a authentication</a>.
+ * */
+public final class OAuth1AToken {
+    private final String consumerKey;
+    private final String accessKey;
+    private final String signatureMethod;
+    private final String signature;
+    private final String timestamp;
+    private final String nonce;
+    /**
+     * Additional Parameters.
+     * */
+    private final Map<String, String> params;
+
+    private OAuth1AToken(String consumerKey, String accessKey, String signatureMethod, String signature,
+                         String timestamp, String nonce, Map<String, String> params) {
+        this.consumerKey = consumerKey;
+        this.accessKey = accessKey;
+        this.signatureMethod = signatureMethod;
+        this.signature = signature;
+        this.timestamp = timestamp;
+        this.nonce = nonce;
+        this.params = params;
```

Comment:
- requireNonNull on all params
- Do some validation on all params and throw IllegalArgumentException
- `ImmutableMap.copyOf()` for `params`